htdocs/install/page_pathsettings.php

```diff
@@ -27,64 +27,68 @@
 
 
 
-class PathStuffController {
-	var $xoopsRootPath = '';
-	var $xoopsTrustPath = '';
-	var $xoopsUrl = '';
+class PathStuffController
+{
+	public $xoopsRootPath = '';
+	public $xoopsTrustPath = '';
+	public $xoopsUrl = '';
 
-	var $validRootPath = false;
-	var $validTrustPath = false;
-	var $validUrl = false;
+	public $validRootPath = false;
+	public $validTrustPath = false;
+	public $validUrl = false;
 
-	var $permErrors = array();
+	public $permErrors = array();
 
-	function PathStuffController() {
-		if (isset( $_SESSION['settings']['ROOT_PATH'] )) {
+	public function PathStuffController()
+	{
```

Comment:
Moving the opening brace to a new line doesn't follow our code standards and adds extras lines to the code base.